src/v2/providers/alerts/crashlytics.ts

```diff
@@ -0,0 +1,325 @@
+import { defineEndpoint, FirebaseAlertData } from '.';
+import { ManifestEndpoint } from '../../../common/manifest';
+import { CloudEvent, CloudFunction } from '../../core';
+import * as options from '../../options';
+
+/** Data */
+/** Generic issue interface */
+interface Issue {
+  id: string;
+  title: string;
+  subtitle: string;
+  appVersion: string;
+}
+/** Payload for a new fatal issue */
+export interface NewFatalIssuePayload {
+  ['@type']: 'com.google.firebase.firebasealerts.CrashlyticsNewFatalIssuePayload';
+  issue: Issue;
+}
+/** Payload for a new non-fatal issue */
+export interface NewNonfatalIssuePayload {
+  ['@type']: 'com.google.firebase.firebasealerts.CrashlyticsNewNonfatalIssuePayload';
+  issue: Issue;
+}
+/** Payload for a regression alert */
+export interface RegressionAlertPayload {
+  ['@type']: 'com.google.firebase.firebasealerts.CrashlyticsRegressionAlertPayload';
+  type: string;
+  issue: Issue;
+  resolveTime: string;
+}
+/** Generic trending issue interface */
+interface TrendingIssueDetails {
+  type: string;
+  issue: Issue;
+  eventCount: number;
+  userCount: number;
+}
+/** Payload for a stability digest */
+export interface StabilityDigestPayload {
+  ['@type']: 'com.google.firebase.firebasealerts.CrashlyticsStabilityDigestPayload';
+  digestDate: string;
+  trendingIssues: TrendingIssueDetails[];
+}
+/** Payload for a velocity alert */
+export interface VelocityAlertPayload {
+  ['@type']: 'com.google.firebase.firebasealerts.VelocityAlertPayload';
+  issue: Issue;
+  createTime: string;
+  crashCount: number;
+  crashPercentage: number;
+  firstVersion: string;
+}
+/** Payload for a new ANR issue */
+export interface NewAnrIssuePayload {
```

Comment:
Nit: I wonder if we should spell out Application Not Responding in the docstring

Reply:
changed and added to other doc strings to make them more descriptive